<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-VC5SNKN" failures="1" tests="9" name="Test" time="6.702" errors="0" timestamp="2022-07-28T22:22:31 GMT-04:00">
  <testcase classname="day32.TestingPriority" name="a" time="0.024"/>
  <testcase classname="day32.TestingPriority" name="b" time="0.002"/>
  <testcase classname="day32.TestingPriority" name="c" time="0.004"/>
  <testcase classname="day32.FirstTestCase" name="test1" time="0.001"/>
  <testcase classname="day32.FirstTestCase" name="test2" time="0.004"/>
  <testcase classname="day32.FirstTestCase" name="test3" time="0.002"/>
  <testcase classname="day32.SecondTest" name="openApp" time="3.561"/>
  <testcase classname="day32.SecondTest" name="login" time="2.761">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at day32.SecondTest.login(SecondTest.java:58)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="day32.SecondTest" name="closeApp" time="0.222"/>
</testsuite> <!-- Test -->
